### PR TITLE
Issue #SC-2086 feat: added dial service deployment config

### PR DESCRIPTION
**Pull request by krgauraw**

### CODE REVIEW

File: ansible/roles/stack-sunbird/defaults/main.yml [hunk 1]
```diff
@@ -204,6 +204,13 @@ taxonomy_service_limit_memory: 512M
 taxonomy_service_reservation_cpu: 0
 taxonomy_service_limit_cpu: 1
 
+# Dial Service Config
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/defaults/main.yml:207]: Please remove this section. This is not used anywhere in the helm chart,

Comment (keshavprasadms) [reply ansible/roles/stack-sunbird/defaults/main.yml:207]: Also please remove these config sections in the same file.
`# Taxonomy Service Config `
`# Search Service`
`# Assessment Service Config`
`# Content Service Config`

**Review by keshavprasadms — CHANGES_REQUESTED** [review]
Please check my comments

File: ansible/roles/stack-sunbird/defaults/main.yml (hunk 1 above)
Comment (krgauraw) [reply ansible/roles/stack-sunbird/defaults/main.yml]: done

**Review by keshavprasadms — APPROVED** [review]

**Pull request merged**